utils/src/main/kotlin/Utils.kt

```diff
@@ -19,12 +19,24 @@
 
 package com.here.ort.utils
 
+import org.apache.logging.log4j.kotlin.KotlinLogger
+import org.apache.logging.log4j.kotlin.loggerOf
+
 import java.io.File
 import java.net.URI
 import java.net.URISyntaxException
 import java.security.Permission
 
-val log = org.apache.logging.log4j.kotlin.logger("ORT")
+/**
+ * Global map of loggers for classes so only one logger needs to be instantiated per class.
+ */
+val loggerOfClass = mutableMapOf<Any, KotlinLogger>()
```

Comment:
The property could be made private.

Reply:
Unfortunately that's not possible (I had tried that) because the getter is inlined (and it has to be `inline` because of `reified`), and then this needs to be `public`, too.

Reply:
True, but then accessing `log` would involve a function call as your `loggerOf` can't be `inline` as that again would require `loggerOfClass` to be public.